packages/client/src/StreamRegistry.ts

```diff
@@ -465,4 +465,19 @@ export class StreamRegistry implements Context {
             })
         ])
     }
+
+    // eslint-disable-next-line class-methods-use-this
+    private streamContractErrorProcessor(err: any, streamId: StreamID, registry: string): never {
+        if (err.errors) {
+            if (err.errors.some((e: any) => {
+                return e.reason && e.reason.code === 'CALL_EXCEPTION'
+            })) {
+                throw new NotFoundError('Stream not found: id=' + streamId)
+            } else {
+                throw new Error(`Could not reach the ${registry} Smart Contract: ${err.errors[0]}`)
+            }
+        } else {
+            throw new Error(err)
+        }
+    }
```

Comment:
If you move this to be a normal function within the file you can get rid of the eslint-disable directive.